package com.training.exception;

public class ExampleOnExceptionHandling4 {

	public static void main(String[] args)throws InterruptedException {
		
		
	System.out.println("statement-1");
	
	/*
	 * try { Thread.sleep(1000); } catch (InterruptedException e) { // TODO
	 * Auto-generated catch block e.printStackTrace(); }
	 */
	Thread.sleep(1000);
	System.out.println("end of the program");
		
		
	}
	
	
	
}
